R/r.R

```diff
@@ -66,6 +66,82 @@ use_test <- function(name = NULL, open = NULL) {
 }
 
 
+#' Automatically rename paired `R/` and `test/` files
+#'
+#' @description
+#' * Moves `R/{old}.R` to `R/{new}.R`
+#' * Moves `tests/testthat/test-{old}.R` to `tests/testthat/test-{new}.R`
+#' * Moves `tests/testthat/test-{old}-*.*` to `tests/testthat/test-{new}-*.*`
+#'   and updates paths in the test file.
+#' * Also removes now unneeded `context()` calls from the test file.
```

Comment:
```suggestion
#' * Removes `context()` calls from the test file, which are unnecessary (and discouraged) as of testthat v2.1.0.
```

Speaking of which, should this PR do anything about putting a minimum version on testthat?

Reply:
testthat 2.1.0 is now a year old, and the version dependencies of suggested packages aren't checked (IIRC)